### PR TITLE
[SYCL] Improve project documentation

### PR DESCRIPTION
**Pull request by vladimirlaz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+## License
+
+Intel Project for LLVM\* technology is licensed under the terms of the Apache
```
Comment (pvchupin) [inline CONTRIBUTING.md:5]: This is copy-pasted wording from intel/llvm project. Could you please fix it so it's relevant to this project.

File: README.txt [hunk 2]
```diff
@@ -6,3 +6,27 @@ Please see the LLVM testing infrastructure guide:
   https://llvm.org/docs/TestSuiteGuide.html
 
 for more information on the contents of this repository.
+
```
Comment (pvchupin) [inline README.txt:9]: I suggest using Readme.md here, will it prevail over Readme.txt in github visualization?
In particular links to License and Contributing are broken below.

File: README.txt [hunk 3]
```diff
@@ -6,3 +6,27 @@ Please see the LLVM testing infrastructure guide:
...
+Intel staging area for llvm-test-suite contribution.
+Home for Intel llvm-test-suite extension targetting DPC++ testing:
+
+* oneAPI Data Parallel C++ compiler - see **intel** branch. More information on
+   oneAPI and DPC++ is available at
+[https://www.oneapi.com/](https://www.oneapi.com/)
+
+## License
+
+See [LICENSE.txt](LICENSE.TXT) for details.
```
Comment (pvchupin) [inline README.txt:21]: broken link

File: README.txt [hunk 4]
```diff
@@ -6,3 +6,27 @@ Please see the LLVM testing infrastructure guide:
...
 for more information on the contents of this repository.
+
+## Introduction
+
+Intel staging area for llvm-test-suite contribution.
+Home for Intel llvm-test-suite extension targetting DPC++ testing:
+
+* oneAPI Data Parallel C++ compiler - see **intel** branch. More information on
+   oneAPI and DPC++ is available at
+[https://www.oneapi.com/](https://www.oneapi.com/)
```
Comment (pvchupin) [inline README.txt:17]: This can be removed.

**Review by pvchupin — COMMENTED** [review]
Please check wording carefully. Made a few comments inlined.

File: README.txt (hunk 2 above)
Comment (vladimirlaz) [reply README.txt:9]: yes. Will restore original README.txt and add README.md specific for our project.

File: README.txt (hunk 3 above)
Comment (vladimirlaz) [reply README.txt:21]: fixed after move to README.md

File: README.txt (hunk 4 above)
Comment (vladimirlaz) [reply README.txt:17]: done

File: CONTRIBUTING.md (hunk 1 above)
Comment (vladimirlaz) [reply CONTRIBUTING.md:5]: done

**Comment by vladimirlaz** [comment]
> Please check wording carefully. Made a few comments inlined.

comments are applied in https://github.com/intel/llvm-test-suite/pull/25

**Pull request merged**